### ISSUE TITLE
Missing 32bit libc.so.6 arch linux

### ISSUE THREAD

**Issue by josefnpat** [ValveSoftware/steam-for-linux#3741]
Today I fired up steam, and the following error occurred:

    seppi@seppi7:~♠ steam
    Repairing installation, linking /home/seppi/.steam/steam to /home/seppi/.local/share/Steam
    Running Steam on arch rolling 64-bit
    STEAM_RUNTIME is enabled automatically
    /media/3TB SATA SEAGATE/Steam/steam.sh: line 347: no match: /media/ubuntu12_32/steam-runtime.tar.xz.part*
    Error: You are missing the following 32-bit libraries, and Steam may not run:
    libc.so.6
    /media/3TB SATA SEAGATE/Steam/steam.sh: line 742: /media/ubuntu12_32/steam: No such file or directory
    seppi@seppi7:~♠ pacman -Ss glibc | grep installed
    core/glibc 2.21-2 (base) [installed]
    multilib/lib32-glibc 2.21-2 [installed]
    seppi@seppi7:~♠ ls /usr/lib32/libc.so.6
    /usr/lib32/libc.so.6

You'll notice I have lib32-glibc installed. I have run steam before on this system!

Any help would be appreciated.

**Comment by gdrewb-valve** [comment]
Is this the same as #3730?

**Comment by josefnpat** [comment]
@gdrewb-valve I'm not sure, as I'm running arch linux.

I saw that issue earlier, but didn't know if it was platform dependent, so I figured I'd make an issue, in case it was.

**Comment by gdrewb-valve** [comment]
It's likely it's a similar thing but I'll leave this one open also.  You could try the Mesa workarounds discussed in the link from #3730 and see if they help.

**Comment by josefnpat** [comment]
I have `lib32-mesa` already installed.

Unless I'm mistaken the workaround is to install `lib32-mesa-libgl` but that conflicts with my `lib32-nvidia-libgl`, which would bomb my nvidia driver stack.

**Comment by gdrewb-valve** [comment]
I was wondering more about the libc portion since Steam isn't finding it even though you appear to have things installed.  You may also have an entirely different issue.

**Comment by josefnpat** [comment]
Alright, I was able to resolve my issue.

Originally I had soft linked my `~/.steam` folder to `/media/3TB SATA SEAGATE/Steam`

I was digging around, and it looks like when steam updated, it didn't recognize the spaces, and changed my soft linking to `/media`.

I have removed:
`~/.steam`
`~/.steampid`
`~/.steampath`
`~/.local/share/Steam/`

And started `steam` up which re-initialized everything. From that, I relinked 

`~/.local/share/Steam/steamapps` to `/media/3TB SATA SEAGATE/Steam/SteamApps` which fixed my issue, and everything runs again.

While this does put ~1GB of data on my primary partition, I'm ok with that as long as the `common` data is not on it, so this is an acceptable solution to me!

Unfortunately I have no idea how to reproduce this issue. I'll leave that up to you if you want to make a new ticket on this subject.

@gdrewb-valve Thanks so much for your time.
